src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days < 7) {
+    return totalCost - 20;
+  }
+
+  return totalCost - 50;
```

Comment:
It is better to use constants for discounts like `SHORT_TERM_DISCOUNT` and `LONG_TERM_DISCOUNT`